pkg/cvo/sync_worker.go

```diff
@@ -181,11 +181,16 @@ type SyncWorker struct {
 	requiredFeatureSet string
 
 	clusterProfile string
+
+	// alwaysEnableCapabilities is a list of cluster capabilities which should
```

Comment:
Why not add these capabilities to SyncWork.Capabilities?  Then you don't have to pass the variable around in the signatures of other functions.